src/blockchain/esplora/reqwest.rs

```diff
@@ -106,19 +106,19 @@ impl Blockchain for EsploraBlockchain {
     }
 
     fn get_tx(&self, txid: &Txid) -> Result<Option<Transaction>, Error> {
-        Ok(self.url_client._get_tx(txid).await?)
+        Ok(await_or_block!(self.url_client._get_tx(txid))?)
```

Comment:
I do not like this usage of `await_or_block` because there is no reason to use `reqwest` in a blocking environment, we have `ureq` for that.